### ISSUE TITLE
argument "clip_length" missing in evaluate_da.py 

### ISSUE THREAD

**Issue by Bang2hen1iu** [xuyu0010/PATAN#3]
There exists an undefined argument in the testing script. 
![image](https://user-images.githubusercontent.com/34578989/173782986-d57810b7-880b-48f3-b54c-e256939e0e3a.png)
I wonder if the value is highly related to the dataset. Is there any detail that I missed in the paper? 
To verify the result with the provided pretrained model on H14-U7, I manually set it as 1 or 5 but dramatic drops happened in testing accuracy.
![image](https://user-images.githubusercontent.com/34578989/173784581-b53f5388-97f2-4e29-a68e-6e9ab79e36e9.png)
Could you help me reproduce the results?

**Comment by xuyu0010** [comment]
Hi, thank you very much for your interest, and thank you very much for pointing out this mistake. It seems that we have accidentally missed this parameter. This value is set to 16 by default (as 16 frames is typically the number of frames used for video/clip feature extraction). We will add this parameter as soon as possible.

Closed as problem solved, do open it if encounter any other problems. Once again thank you very much for pointing it out!

**Comment by Bang2hen1iu** [comment]
Thank you for your reply. I have another question. The pretrained model provided for H14-U7 seems not to work. I follow the instruction and put it in the subfolder .../exps/models/. The results are still very low even though the clip_length changed to 16. It seems that this parameter doesn't affect the testing result as I got exactly the same top-1 testing accuracy. Are there any details that need to be modified in the codes?

**Comment by xuyu0010** [comment]
I am not very sure about your other settings (including the PyTorch version and Cuda versions). We have replicated our results several times with this code so should be correct. I suggest you try to check with DANN first (by using the --da-method DANN option) and see if you can reach our reported result of around 74-75%. If not, you may downgrade your PyTorch version to 1.2.0. This code was writen well over a year ago so its compatibility with current version of PyTorch is not guaranteed.

**Comment by Bang2hen1iu** [comment]
OK. My Pytorch version is 1.8 and this might be the reason for this problem. I'll try to downgrade it and test as you suggest. Thanks a lot for your kindly reply. ^_^

**Comment by Bang2hen1iu** [comment]
Demo weights work pretty well after shifting codes on a server with PyTorch 1.2.0 (CUDA10.2) and RTX2080Ti.  Thanks for sharing all the information. 
Besides, when check with DANN, the inputs are set as zero tensors:
![image](https://user-images.githubusercontent.com/34578989/175246950-053d181d-a585-4ea6-b360-5e091c4ff3f1.png)
I guess that the inputs should pass into the ad_net after a grl operation. 
After modifying the code I rerun the training script with DANN but the result is still weired, loss_aux_clip floating between 2.6-3.0 or higher. The model cannot fit the training set which result in a training acc on the target with less than 50% 
May I ask for a training log on U14-H7 to help me check if there is a bug?

**Comment by xuyu0010** [comment]
Hi, thank you for your sharing progress. We will look into the code and see if there is anything missed when we reorganized our code back then, sorry for the mistakes! As I actually graduated from school so I may need some time to retrieve the training logs. I will open a new issue on the logs to remind myself.

**Comment by xuyu0010** [comment]
You may refer to #4 for the logs. Thank you for your patience!

**Comment by Bang2hen1iu** [comment]
This is the log file of my own trying. 
[PATAN-0623.log](https://github.com/xuyu0010/PATAN/files/8967449/PATAN-0623.log)
